### DISCUSSION TITLE
Not redirecting after Processing application

### DISCUSSION THREAD

**Discussion by Kira-Pgr** [community/community#65802] [GitHub Education]
### Hi everyone,

On `Upload proof of your academic status` page
![image](https://github.com/community/community/assets/36188023/5a689952-ad70-4b2b-8bc6-f5daff292f17)
After it goes to 100%, it still goes to the same page.

**Comment by arafajs** [comment]
Try a different browser.

**Comment by JittoJoseph** [comment]
i got the same problem

**Comment by Kira-Pgr** [comment]
I tried several browsers, even devices, still not working
